server/compose/types/namespace.go

```diff
@@ -15,6 +15,7 @@ type (
 		Slug    string        `json:"slug"`
 		Enabled bool          `json:"enabled"`
 		Meta    NamespaceMeta `json:"meta"`
+		Blocks  PageBlocks    `json:"blocks"`
```

Comment:
I'm not really a fan of reusing this struct like so.
I'd prefer it sticks consistent as we have it with [page layouts](https://github.com/cortezaproject/corteza/blob/2023.9.x/server/compose/types/page_layout.go#L31)